modules/openapi-generator/src/main/resources/typescript/auth/auth.mustache

```diff
@@ -123,13 +123,14 @@ export type ApiKeyConfiguration = string;
 export type HttpBasicConfiguration = { "username": string, "password": string };
 export type HttpBearerConfiguration = { tokenProvider: TokenProvider };
 export type OAuth2Configuration = { accessToken: string };
+export type HttpSignatureConfiguration = unknown; // TODO: Implement
```

Comment:
I am not actually sure what this has to look like. Currently it's not supported as we don't have it implemented. This fix is to produce valid syntactic code, the implementation is probably a bit more involved.
You can see the fix it produces below in `samples/openapi3/client/petstore/typescript/builds/explode-query/auth/auth.ts`.
And how it is currently broken (syntactically invalid TS), here: https://github.com/OpenAPITools/openapi-generator/blob/ce09134b4877f92fea4e336a2998f69e2cc42ed5/samples/openapi3/client/petstore/typescript/builds/explode-query/auth/auth.ts#L164-L166